internal/operators/manager_test.go

```diff
@@ -68,6 +68,21 @@ var _ = Describe("Operators manager", func() {
 		Expect(manifests).To(HaveKey(ContainSubstring("openshift-lso")))
 	})
 
+	It("should generate CNV and LSO manifests when CNV operator is enabled", func() {
```

Comment:
That should probably be OCS and LSO whe CNV is enabled. Unless the requirement of CNV to depend on OCS is no longer valid.

Reply:
We do not depend on OCS anymore.